src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write code here
+const employees = document.querySelectorAll('li');
+
+function convertSalarytoNum(str) {
+  return +(str.split(',').join('').slice(1));
+}
+
+function sortList([...list]) {
+  return list.sort((a, b) => (
```

Comment:
```suggestion
function sortList(list) {
  return [...list].sort((a, b) => (
```
If you want to create a copy, then just do it inside the function (or before, in employees const), but not with the parameter directly.

Reply:
+